### ISSUE TITLE
Add controlled realearn group/preset to controller projection

### ISSUE THREAD

**Issue by jackmau** [helgoboss/realearn#187]
Out of the million FR which could be imagined in terms of addition to the _awesome_ controller projection functionality, this is, in my use case, at the very top. It would be nice to have a custom _widget_ in the companion app, one can put somewhere in the editor to get name of the current active realearn group/preset. This is currently achievable by adding an _imaginary_ virtual controller and creating some dummy assignment with user-defined text, but I'd like a more intuitive native implementation

**Comment by helgoboss** [comment]
Interesting workaround :)

It's not exactly what you requested but you can see the name of the active main preset in the toolbar, so if you don't hide it (by tapping the background), you can always see the active preset.

At the moment, ReaLearn doesn't have the concept of a single "Active group". There can be many active groups, so I think this part of the request should be given some more thought.

**Comment by jackmau** [comment]
> Interesting workaround :)

In my long history of hacking around realearn this is not even the worst XD

> It's not exactly what you requested but you can see the name of the active main preset in the toolbar, so if you don't hide it (by tapping the background), you can always see the active preset.

I hadn't noticed, I will test.

> At the moment, ReaLearn doesn't have the concept of a single "Active group". There can be many active groups, so I think this part of the request should be given some more thought.

Very fair point, in my mind group=page=some-internal-realearn-parameter-value, but that's definitely not how things are. Maybe the solution would be to be able to visualise realearn parameters instead, of course being able to assign each parameter value to a more meaningful text description...not sure how it would work GUI-wise

**Comment by helgoboss** [comment]
Okay, let me know when your idea takes shape. I will put this in a waiting state until then (there are many FRs now, need to sort a bit).

**Comment by jackmau** [comment]
Yeah this is far from urgent, controller projection is still in its early days and I imagine other features (like current value) may be more user-requested

**Comment by pyrannial** [comment]
Yes, this would be a very useful feature for me too  
https://forum.cockos.com/showpost.php?p=2433568&postcount=1190
